src/app/components/Figure/Image/index.amp.jsx

```diff
@@ -0,0 +1,6 @@
+import 'styled-components';
+import { StyledImg } from '.';
+
+const AmpImg = StyledImg.withComponent('amp-img');
```

Comment:
V4 of Styled Components adds the `as` prop https://www.styled-components.com/docs/api#as-polymorphic-prop. We should use this in future. 

However, since Simorgh is still using V3 of Styled Components, we cannot yet use the `as` prop and have to use the `withComponent` syntax instead.

Reply:
Good news! https://github.com/BBC-News/simorgh/blob/latest/package.json#L93

Reply:
Fair point - should we move to Psammead first and make use of latest v4 functionality, rather than update in Simorgh and then upgrade in the transfer?

Reply:
The `withComponent` syntax still works in v4, so it'll work in Psammead.